var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -39,24 +39,33 @@ class Legion(CMakePackage):
 
     variant('mpi', default=True,
             description='Build on top of mpi conduit for mpi inoperability')
+    variant('ibv', default=False,
+            description='Build on top of ibv conduit for InfiniBand support')
     variant('shared', default=True, description='Build shared libraries')
     variant('hdf5', default=True, description='Enable HDF5 support')
 
-    depends_on("cmake@3.1:", type='build')
+    depends_on("cmake@3.12.4", type='build')
     depends_on("gasnet~aligned-segments~pshm segment-mmap-max='16GB'", when='~mpi')
     depends_on("gasnet~aligned-segments~pshm segment-mmap-max='16GB' +mpi", when='+mpi')
+    depends_on("gasnet~aligned-segments~pshm segment-mmap-max='16GB' +ibv", when='+ibv')
     depends_on("hdf5~mpi", when='+hdf5')
 
     def cmake_args(self):
         options = [
+            '-DCMAKE_BUILD_TYPE=Debug',
             '-DLegion_USE_GASNet=ON',
+            '-DLEGION_USE_CUDA=OFF',
+            '-DLEGION_USE_OPENMP=OFF',
             '-DLegion_BUILD_EXAMPLES=ON',
+            '-DCMAKE_CXX_FLAGS=-DPRIVILEGE_CHECKS -DDEBUG_REALM -DDEBUG_LEGION -DBOUNDS_CHECKS -DENABLE_LEGION_TLS -ggdb',
```

Comment:
Adding debug flags should only be done for the debug build.

Reply:
ohh yeah I can remove that line...
how do I separate the one long line of CXX flags?

Reply:
That would work, you could also make a python list like

```
['-DPRIVILEGE_CHECKS', '-DDEBUG_REALM', ...]
```

and then `' '.join()` it.